src/components/Navbar.jsx

```diff
@@ -5,12 +5,17 @@ import '../index.css';
 const Navbar = () => (
   <div className="nav-header">
     <nav className="nav-menu">
-      <h1 className="text-xl font-bold text-blue">Bookstore CMS</h1>
-      <NavLink to="/">Books</NavLink>
-      <NavLink to="/Categories">Categories</NavLink>
+      <h1 className="text-3xl font-bold text-blue">Bookstore CMS</h1>
+      <NavLink className="NavLink" to="/">Books</NavLink>
+      <NavLink className="NavLink" to="/Categories">Categories</NavLink>
     </nav>
 
-    <div className="userLogo text-blue"><p>userlogo</p></div>
+    <div className="userLogo border h-12 w-12 mr-12">
+      <svg xmlns="http://www.w3.org/2000/svg" className="ionicon fill-blue" viewBox="0 0 512 512">
+        <path d="M258.9 48C141.92 46.42 46.42 141.92 48 258.9c1.56 112.19 92.91 203.54 205.1 205.1 117 1.6 212.48-93.9 210.88-210.88C462.44 140.91 371.09 49.56 258.9 48zm126.42 327.25a4 4 0 01-6.14-.32 124.27 124.27 0 00-32.35-29.59C321.37 329 289.11 320 256 320s-65.37 9-90.83 25.34a124.24 124.24 0 00-32.35 29.58 4 4 0 01-6.14.32A175.32 175.32 0 0180 259c-1.63-97.31 78.22-178.76 175.57-179S432 158.81 432 256a175.32 175.32 0 01-46.68 119.25z" />
+        <path d="M256 144c-19.72 0-37.55 7.39-50.22 20.82s-19 32-17.57 51.93C191.11 256 221.52 288 256 288s64.83-32 67.79-71.24c1.48-19.74-4.8-38.14-17.68-51.82C293.39 151.44 275.59 144 256 144z" />
+      </svg>
+    </div>
   </div>
```

Comment:
- [ ] Good job adding the NavBar to you web application 👏 , It could be better if the header elements are aligned as they are in the provided [design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378). Kindly fix that as described below 🖐️ 

### Your design
![Capture](https://github.com/mhnazary/bookstore/assets/80458199/fb58470b-4ed2-49ac-aaff-7bf37ea09145)
 
### Design template
![Capture0](https://github.com/mhnazary/bookstore/assets/80458199/25e3d327-ac40-455d-ac8d-e14d4acfa335)